install.sh

```diff
@@ -10,6 +10,29 @@ case $OSTYPE in
     ;;
 esac
 
+BACKUP_FILE=$CONFIG_FILE.bak
+
+if [ -e $HOME/$BACKUP_FILE ]; then
+  echo "Backup file already exist. Make sure to backup your .bashrc before running this installation." >&2
+     while true
+    do
+      read -p "Would you like to overwrite the existing backup? this will delete your existing backup file [Y/N] " RESP
+      case $RESP
+      in
+      [yY])
+          break
+        ;;
+      [nN])
```

Comment:
Check out the way the default value is handled here: https://github.com/Bash-it/bash-it/blob/master/install.sh#L50 

In case someone simply presses Enter at the prompt, _N_ is assumed as a default value.